### PR TITLE
Update data in seed ,change GetUserTestByPassedTestIdAsync and DTO

### PR DESCRIPTION
**Pull request by ElenaBurak**

### CODE REVIEW

File: exalearn-api/ExaLearn.Bl/DTO/PassedTestForCheckDTO.cs [hunk 1]
```diff
@@ -2,7 +2,9 @@
 {
     public class PassedTestForCheckDTO
     {
-        public QuestionDTO[] Questions { get; set; }
+        public int Id { get; set; }
+
+        public int AssessmentAllTest { get; set; }
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/DTO/PassedTestForCheckDTO.cs]: what is this assessment all test?

Comment (romanstat) [reply exalearn-api/ExaLearn.Bl/DTO/PassedTestForCheckDTO.cs]: generalassessment?

**Comment by romanstat** [comment]
remove FindByNameAsync method in repository

File: exalearn-api/ExaLearn.Bl/DTO/PassedTestForCheckDTO.cs (hunk 1 above)
Comment (ElenaBurak) [reply exalearn-api/ExaLearn.Bl/DTO/PassedTestForCheckDTO.cs]: It is name, which asks for front

**Review by romanstat — APPROVED** [review]

**Review by artemgael96 — APPROVED** [review]

**Pull request merged**